utils/analytics.py

```
@@ -0,0 +1,74 @@
+        )
+    send_ga_event.delay(client_id, _serialize_events(events))
+
+
+@shared_task(name="send_ga_event")
+def send_ga_event(client_id: str, events: list[Event]):
+    base_url = "https://www.google-analytics.com/mp/collect"
+    params = {"measurement_id": settings.GA_MEASUREMENT_ID, "api_secret": settings.GA_API_SECRET}
+    url = f"{base_url}?{urlencode(params)}"
+    payload = {"client_id": client_id, "events": events}
+
+    response = requests.post(url, json=payload)
```

Ruff (0.14.5)

Probable use of `requests` call without timeout

(S113)